k-distribution/tests/regression-new/coverage-poly/test.k

```diff
@@ -0,0 +1,7 @@
+module TEST
+  imports K-REFLECTION
+
+  syntax Foo ::= parseFoo() [function]
+  rule parseFoo() => #parseKORE("Lblfoo{}()")
+  syntax Foo ::= foo() [symbol]
+endmodule
```

Comment:
Is this test helpful other than 'it doesn't crash'?

Reply:
Yes. The test got stuck before I made these changes, because when --coverage was passed to kompile, the rule for `parseFoo` was being instrumented in a manner that caused the rule to not apply.